src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -589,12 +589,14 @@ def preview_osdspecs(self,
     def remove_osds(self, osd_ids: List[str],
                     replace: bool = False,
                     force: bool = False,
-                    zap: bool = False) -> OrchResult[str]:
+                    zap: bool = False,
+                    no_destroy: bool = False) -> OrchResult[str]:
```

Comment:
looks like this requires this arg being added to the rook orchestrator as well (for make check to pass)
```
rook/__init__.py:5: note: In module imported here:
rook/module.py: note: In member "remove_osds" of class "RookOrchestrator":
rook/module.py:628: error: Signature of "remove_osds" incompatible with supertype "Orchestrator"
```